test-packages/memory-tests/sdk-v1/test-script.js

```diff
@@ -0,0 +1,93 @@
+const memwatch = require('node-memwatch-new');
+const { writeFileSync } = require('fs');
+const { resolve } = require('path');
+const { TestEntity, TestEntity50Prop } = require('./test-service');
```

Comment:
Question: How does this require work - there is no index.js or file/folder `test-service` next to the test-script file?